lib/matplotlib/backend_bases.py

```diff
@@ -2221,24 +2213,22 @@ def on_press(event):
 
     def mpl_disconnect(self, cid):
         """
-        Disconnect callback id cid
+        Disconnect the callback with id *cid*.
 
         Examples
         --------
-        Usage::
-
+        ::
             cid = canvas.mpl_connect('button_press_event', on_press)
-            #...later
+            # ... later
             canvas.mpl_disconnect(cid)
         """
         return self.callbacks.disconnect(cid)
 
     def new_timer(self, *args, **kwargs):
         """
-        Creates a new backend-specific subclass of
-        :class:`backend_bases.Timer`. This is useful for getting periodic
-        events through the backend's native event loop. Implemented only for
-        backends with GUIs.
+        Creates a new backend-specific subclass of `.Timer`.  This is useful
```

Comment:
```suggestion
        Create a new backend-specific subclass of `.Timer`.  This is useful
```
plus empty line after the first sentence.